.github/workflows/e2e_android.yml

```diff
@@ -33,19 +33,19 @@ env:
   # Filepath for firebase distribution key. Also used by fastlane
   FIREBASE_DIST_KEY_FILEPATH: ${{ secrets.FIREBASE_DIST_KEY_FILEPATH }}
   # Slack API token used by fastlane
-  # SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
+  SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
 
 jobs:
-  # start_slack_thread:
-  #   name: Start Slack thread
-  #   uses: ./.github/workflows/start_slack_thread.yml
-  #   secrets: inherit
-  #   with:
-  #     channel_name: va-mobile-build-alerts
-  #     message: 'E2E Android build process starting. Please see :thread: for results. This process may take a while.'
+  start_slack_thread:
```

Comment:
Since this workflow runs on dependabot PRs now, we'll need to add logic to only have this run when the trigger is a cron:
``` yml
if: github.event_name == 'schedule'
```